webp/view/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ add_library(app-glue STATIC
 include_directories(${webp_src_dir}/examples
                     ${webp_src_dir}/src)
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c99")
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Werror -Wno-reorder")
```

Comment:
that might be a too much a ask for developers to know the sequence of initialization: but it  definitely could be fixed.  Like to addressed together with other samples ( maybe this warning is in most samples I feel )